### ISSUE TITLE
Vite 5.3.0 causing build failure

### ISSUE THREAD

**Issue by simonwfarrow** [likec4/likec4#765]
Hi,

The package.json has the vite dependency set as ^5.2.11, this causes it to use the latest 5.3.0 version , which produces the below build error. I can workaround the problem by setting my local package to have vite@5.2.13

```
4:39:48 PM c4:lsp  INFO version 1.1.1                                                                                                            4:39:48 PM
4:39:48 PM c4:lsp  INFO layout wasm                                                                                                              4:39:48 PM
4:39:48 PM c4:lsp  INFO workspace: /Users/e5591703/Projects/mdl/src                                                                              4:39:48 PM
4:39:50 PM c4:lsp  INFO workspace: found 39 sources                                                                                              4:39:50 PM
4:39:50 PM c4:lsp  INFO workspace: ✓ built 75 views                                                                                              4:39:50 PM
4:39:50 PM c4:vite INFO likec4 app root /Users/e5591703/.npm/_npx/76390c0e3ff8279b/node_modules/likec4/dist/__app__                              4:39:50 PM
4:39:50 PM c4:vite INFO output /Users/e5591703/Projects/mdl/dist                                                                                 4:39:50 PM
4:39:50 PM c4:lib INFO outDir /var/folders/ff/mw3p2kbx0h19mprhyfd00l6w0000gp/T/.likec4-public-NSXFE7                                             4:39:50 PM
4:39:50 PM c4:lib INFO vite v5.3.0 building for production...                                                                                    4:39:50 PM
transforming (1) src/lib/webcomponent.mjs4:39:51 PM c4:lib INFO generating virtual:likec4/views                                                                                           4:39:51 PM
4:39:55 PM c4:lib INFO ✓ 2 modules transformed.                                                                                                  4:39:55 PM
[4:39:55 PM] 4:39:55 PM c4:lib INFO ../../../../../../../../../var/folders/ff/mw3p2kbx0h19mprhyfd00l6w0000gp/T/.likec4-public-NSXFE7/likec4-views.js  1,474.68 kB │ gzip: 322.65 kB
4:39:55 PM c4:lib INFO ✓ built in 5.07s                                                                                                          4:39:55 PM
4:39:55 PM c4:vite INFO vite v5.3.0 building for production...                                                                                   4:39:55 PM
4:39:56 PM c4:vite INFO ✓ 5 modules transformed.                                                                                                 4:39:56 PM

 ERROR  4:39:56 PM c4:vite ERROR x Build failed in 664ms                                                                                         4:39:56 PM


[4:39:56 PM]  ERROR  [commonjs--resolver] ../../../.npm/_npx/76390c0e3ff8279b/node_modules/likec4/dist/__app__/src/main-DvKdwZif.js (37785:49): Unexpected token string literal (/mmd, "/mmd"). Expected yield, an identifier, [ or {
file: /Users/e5591703/.npm/_npx/76390c0e3ff8279b/node_modules/likec4/dist/__app__/src/main-DvKdwZif.js:37785:49

37783:   () => import("./view._viewId.mmd.lazy-DUPqzS9E.js").then((d2) => d2.Route)
37784: ), ViewViewIdDotLazyRoute = ViewViewIdDotLazyImport.update({
37785:   path: "/dot",
                      ^
37786:   getParentRoute: () => ViewViewIdRoute
37787: }).lazy(


  file: /Users/e5591703/.npm/_npx/76390c0e3ff8279b/node_modules/likec4/dist/__app__/src/main-DvKdwZif.js:37785:49
  
  37783:   () => import("./view._viewId.mmd.lazy-DUPqzS9E.js").then((d2) => d2.Route)
  37784: ), ViewViewIdDotLazyRoute = ViewViewIdDotLazyImport.update({
  37785:   path: "/dot",
  ^
  37786:   getParentRoute: () => ViewViewIdRoute
  37787: }).lazy(
  
  RollupError: Unexpected token string literal (/mmd, "/mmd"). Expected yield, an identifier, [ or {
  at getRollupError (/Users/e5591703/.npm/_npx/76390c0e3ff8279b/node_modules/rollup/dist/es/shared/parseAst.js:396:41)
  at ParseError.initialise (/Users/e5591703/.npm/_npx/76390c0e3ff8279b/node_modules/rollup/dist/es/shared/node-entry.js:11332:28)
  at convertNode (/Users/e5591703/.npm/_npx/76390c0e3ff8279b/node_modules/rollup/dist/es/shared/node-entry.js:13082:10)
  at convertProgram (/Users/e5591703/.npm/_npx/76390c0e3ff8279b/node_modules/rollup/dist/es/shared/node-entry.js:12399:12)
  at Module.setSource (/Users/e5591703/.npm/_npx/76390c0e3ff8279b/node_modules/rollup/dist/es/shared/node-entry.js:14246:24)
  at async ModuleLoader.addModuleSource (/Users/e5591703/.npm/_npx/76390c0e3ff8279b/node_modules/rollup/dist/es/shared/node-entry.js:18892:13)
```

**Comment by davydkov** [comment]
Thank you for reporting.
I will look into it 👀

**Comment by mcpride** [comment]
Unfortunately it doesn't seem to be fixed in 1.2.1:

```
Using docker image sha256:31791b67055ee80a83cb51552dc549f73f4217b422dc8eff27f331343d13844d for mcr.microsoft.com/playwright:v1.44.1-jammy with digest mcr.microsoft.com/playwright@sha256:b586583ac9d03f4847bedf3c66556fc9a2aaca3ecffb079907a3073cd0fbb103 ...
$ npm install --global likec4@$LIKEC4_VERSION
added 92 packages in 7s
10 packages are looking for funding
  run `npm fund` for details
$ likec4 build -o ./dist --base $BASE_URL_JOB/
[log] INFO version 1.2.1
[log] INFO layout wasm

---- snip -----

[log] INFO outDir /tmp/.likec4-public-XXXXXXDDl8aG
[log] INFO vite v5.3.0 building for production...
[log] INFO transforming...
[log] INFO generating virtual:likec4/views
[log] INFO ✓ 2 modules transformed.
[log] INFO rendering chunks...
[log] INFO computing gzip size...
[log] INFO ../../../../../../tmp/.likec4-public-XXXXXXDDl8aG/likec4-views.js  1,903.40 kB │ gzip: 367.77 kB
[log] INFO ✓ built in 6.30s
[log] INFO vite v5.3.0 building for production...
[log] INFO transforming...
[log] INFO ✓ 5 modules transformed.
[error] ERROR x Build failed in 573ms
[error] [commonjs--resolver] ../../../../../../../usr/lib/node_modules/likec4/dist/__app__/src/main-yVRq2dIl.js (36966:49): Unexpected token `string literal (/mmd, "/mmd")`. Expected yield, an identifier, [ or {
file: /usr/lib/node_modules/likec4/dist/__app__/src/main-yVRq2dIl.js:36966:49
36964:   () => import("./view._viewId.mmd.lazy-By6xPESQ.js").then((d) => d.Route)
36965: ), ViewViewIdDotLazyRoute = ViewViewIdDotLazyImport.update({
36966:   path: "/dot",
                      ^
36967:   getParentRoute: () => ViewViewIdRoute
36968: }).lazy(
  file: /usr/lib/node_modules/likec4/dist/__app__/src/main-yVRq2dIl.js:36966:49
  
  36964:   () => import("./view._viewId.mmd.lazy-By6xPESQ.js").then((d) => d.Route)
  36965: ), ViewViewIdDotLazyRoute = ViewViewIdDotLazyImport.update({
  36966:   path: "/dot",
  ^
  36967:   getParentRoute: () => ViewViewIdRoute
  36968: }).lazy(
  
  RollupError: Unexpected token `string literal (/mmd, "/mmd")`. Expected yield, an identifier, [ or {
  at getRollupError (/usr/lib/node_modules/likec4/node_modules/rollup/dist/es/shared/parseAst.js:396:41)
  at ParseError.initialise (/usr/lib/node_modules/likec4/node_modules/rollup/dist/es/shared/node-entry.js:11332:28)
  at convertNode (/usr/lib/node_modules/likec4/node_modules/rollup/dist/es/shared/node-entry.js:13082:10)
  at convertProgram (/usr/lib/node_modules/likec4/node_modules/rollup/dist/es/shared/node-entry.js:12399:12)
  at Module.setSource (/usr/lib/node_modules/likec4/node_modules/rollup/dist/es/shared/node-entry.js:14246:24)
  at async ModuleLoader.addModuleSource (/usr/lib/node_modules/likec4/node_modules/rollup/dist/es/shared/node-entry.js:
```

**Comment by simonwfarrow** [comment]
Thank you for the quick fix!
